src/chicory/broker/rabbitmq.py

```
@@ -0,0 +1,782 @@
+
+        # Publish with per-message TTL
+        await channel.default_exchange.publish(
+            aio_pika.Message(
+                body=data,
+                content_type="application/json",
+                delivery_mode=aio_pika.DeliveryMode.PERSISTENT
+                if self.durable_queues
+                else aio_pika.DeliveryMode.NOT_PERSISTENT,
+                message_id=message.id,
+                timestamp=datetime.now(UTC),
+                # TODO @dadodimauro: Verify if expiration should be in ms  # noqa: TD003
```

Ruff (0.14.10)

Unused `noqa` directive (non-enabled: `TD003`)

Remove unused `noqa` directive

(RUF100)